lichess-bot.py

```diff
@@ -440,7 +441,7 @@ def get_greeting(greeting):
                                                     start_time,
                                                     move_overhead)
                     else:
-                        engine.add_null_comment()
+                        engine.add_comment(comment, board)
```

Comment:
Since there is now always a comment from every move (including `None` in the case of a local book move), this line can be de-indented and the `else:` deleted. The `self.add_comment()` can be removed from `EngineWrapper.search()`, as well.